.gitattributes

```
@@ -2,4 +2,5 @@
 /.github export-ignore
 /.gitattributes export-ignore
 /scripts export-ignore
-*.css linguist-detectable=false
+/src/Illuminate/Foundation/** linguist-vendored
+/tests/Assets/__fixtures__/** linguist-detectable=false
```

```suggestion
/tests/Assets/__fixtures__/** -linguist-detectable
```
sauce: https://github.com/github-linguist/linguist/blob/master/docs/overrides.md